### PR TITLE
[Test] Chore: refactor `e2eTests` module

### PR DESCRIPTION
**Pull request by yasinmiran**
- [x] Replaced the env vars with static variables.
- [x] Added README file
- [x] Added subtask `gradle assemble` in the `start-docker-containers`

[WIP]: The following tasks will be committed in a separate PR:-
- [ ] ~Remove the redundant certificate file paths~
- [ ] ~Test without TLS~

### CODE REVIEW

File: e2eTests/src/test/java/IngestionTest.java [hunk 1]
```diff
@@ -190,7 +201,7 @@ private void triggerIngestMessageFromCEGA()
     String message =
         String.format(
             "{\"type\":\"ingest\",\"user\":\"%s\",\"filepath\":\"/p11-dummy@elixir-europe.org/files/%s\"}",
```
Comment (joshbaskaran) [inline e2eTests/src/test/java/IngestionTest.java:203]:
```suggestion
    String message = """
    {
        "type": "ingest",
        "user": "%s",
        "filepath": "/p11-dummy@elixir-europe.org/files/%s"
    }
    """.formatted(EGA_BOX_USERNAME, encFile.getName());
```
Purely for the sake of readability and maintainability.

File: e2eTests/src/test/java/IngestionTest.java [hunk 2]
```diff
@@ -225,7 +234,7 @@ private void triggerAccessionMessageFromCEGA()
     String message =
         String.format(
             "{\"type\":\"accession\",\"user\":\"%s\",\"filepath\":\"/p11-dummy@elixir-europe.org/files/%s\",\"accession_id\":\"%s\", \"decrypted_checksums\": [ { \"type\":\"sha256\", \"value\": \"%s\" },{\"type\":\"md5\", \"value\": \"%s\"} ] }",
```
Comment (joshbaskaran) [inline e2eTests/src/test/java/IngestionTest.java:236]: Probably best to format all JSON strings using formatter. Avoids accidental errors.

**Review by joshbaskaran — CHANGES_REQUESTED** [review]

**Comment by joshbaskaran** [comment]
We should also consider merging e2eTest-setup branch into main. And setting up PR's directly on main.

File: e2eTests/src/test/java/IngestionTest.java (hunk 1 above)
Comment (yasinmiran) [reply e2eTests/src/test/java/IngestionTest.java]: Addressed in [2a59f2a](https://github.com/ELIXIR-NO/FEGA-Norway/pull/124/commits/2a59f2af44be312d5c2495b2cf17e58fd5b77791)

File: e2eTests/src/test/java/IngestionTest.java (hunk 2 above)
Comment (yasinmiran) [reply e2eTests/src/test/java/IngestionTest.java]: Addressed in [2a59f2a](https://github.com/ELIXIR-NO/FEGA-Norway/pull/124/commits/2a59f2af44be312d5c2495b2cf17e58fd5b77791)

**Review by Parisa68 — APPROVED** [review]

**Pull request merged**